services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/_constants.py

```diff
@@ -0,0 +1,10 @@
+from typing import Final
+
+_GIGA: Final[int] = int(1e9)
+_DELAY_NS: Final[int] = 5 * _GIGA
```

Comment:
this GIGA variable feels strange, you want to express that your delay is 5 SECONDS but the variable delay has to be in nanoseconds. I think it would read better as
```python
_SECOND = 1_000_000_000 # expressed in nano-seconds

DOCKER_CONTAINER_SPEC_RESTART_POLICY_DEFAULTS = {
    "Condition": "on-failure",
    "MaxAttempts": 100,
    "Delay": 5*_SECOND,
}
```

Reply:
agree with @pcrespov it is weird. also I still do not get why you just do not set the Maxattempts to 0, which means it will try indefinitely. setting it to 100 does make no sense for me.